README.md

```diff
@@ -66,6 +66,9 @@ any suggestions! All of the software here is written by students.
      [Node](https://nodejs.org/en/), [MongoDB](https://www.mongodb.com), and
      [Yarn](https://yarnpkg.com), then uses these programs to install the
      packages we use and set up a local database.
+   - If you have a problem that looks like
```

Comment:
Are you talking about this part:
"@lenhsherr I think I figured it out. When I upgraded to Mac OS 10.13, the upgrade messed up my homebrew installation, including a bunch of header files under /usr/local/include directory. I renamed this directory and don't see any more compilation errors. So I'll have to remove this directory and reinstall my brew packages."

Can you directly include what he says to do? But still keep the link for reference

Reply:
Ok! I just changed it.